GNUmakefile

```diff
@@ -57,4 +60,42 @@ ifeq (,$(wildcard $(GOPATH)/src/$(WEBSITE_REPO)))
 endif
 	@$(MAKE) -C $(GOPATH)/src/$(WEBSITE_REPO) website-provider-test PROVIDER_PATH=$(shell pwd) PROVIDER_NAME=$(PKG_NAME)
 
+docker-build:
+	@docker build -t $(DOCKER_IMAGE_NAME) .
+
+docker-start:
```

Comment:
`docker-start` and `docker-run` seem very similar from a naive perspective. I know they're documented in your confluence page (I looked there to see if that explained the difference), but could we distinguish them a bit better either name wise, or by at least adding a comment above them to tell them apart?

Reply:
The names make sense mainly because of the underlying docker commands being executed. I'll add comments explaining the difference.